src/components/Details/DetailsComponent.tsx

```diff
@@ -196,6 +209,103 @@ export const DetailsComponent = () => {
     }
   }
 
+  function deleteElementProperty(name: string) {
+    let query = '';
+    if (selectedHeader === "Node") {
+      query = `g.V(${selectedId})`;
+      deleteVertexProperty(name)
+        .then(() => {
+          return axios
+            .post(
```

Comment:
Large amount of duplicate code with line 145. I'd extract the code into a function so it's not duplicated.